test/zencode_petition/run.sh

```diff
@@ -9,6 +9,12 @@
 if ! test -r ../utils.sh; then
 	echo "run executable from its own directory: $0"; exit 1; fi
 . ../utils.sh
+
+is_cortexm=false
```

Comment:
Can you eliminate (or reduce) the use of `is_cortexm`?

Reply:
I think it is better to keep the "is_cortexm" variable. Since the "Z" is the function, the "$1" will be overridden when passing the argument.